plugins/google_directory/unit_test/responses/suspend_user.json.resp

```diff
@@ -1,4 +1,4 @@
-{
+ {
```

Comment:
Apologies if this is pedantic. 
Will this still work with the extra space here?

Reply:
It worked but you're right that it didn't have to look this way 😃